src/handlers/int-test-support/helpers/s3Helper.ts

```diff
@@ -132,6 +136,76 @@ const deleteS3Objects = async (
   );
 };
 
+/* Deletes s3 objects by prefixes in batches */
+export const batchDeleteS3ObjectsByPrefixesInBatchBasic = async (
+  bucketName: string,
+  prefixesToDelete: string[],
+  batchSize: number
```

Comment:
Why is `batchSize` a parameter? Does any user of this function care what batch size the delete uses? Could we just always use the maximum batch size?